src/main/java/org/jabref/gui/groups/GroupDialogView.java

```diff
@@ -29,7 +29,7 @@
 public class GroupDialogView extends BaseDialog<AbstractGroup> {
 
     // Basic Settings
-    @FXML private TextField nameField;
+   @FXML private TextField nameField;
```

Comment:
Identation is off. Please restore.

In IntelliJ you have a command to autoformat code according to our standards, by selecting the code and pressing `Ctrl` + `Alt` + `L`. I don't know if vscode does have this option too. Maybe it's worth a few minutes researching. Saves a lot of time.

About our codingstandards, we use checkstyle to keep our standards coherent. You can install this for vscode here: https://marketplace.visualstudio.com/items?itemName=shengchen.vscode-checkstyle
I did not use checkstyle with vscode, so I'm no help in configuring it, but it should be straightforward with the documentation of the plugin.